src/main/java/com/bookstore/controller/BookController.java

```diff
@@ -32,4 +34,16 @@ public BookDto getBookById(@PathVariable Long id) {
     public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
+
+    @PutMapping("{id}")
+    public BookDto updateBookById(@PathVariable Long id,
+                                  @RequestBody CreateBookRequestDto createBookRequestDto) {
+        return bookService.updateById(id, createBookRequestDto);
+    }
+
+    @DeleteMapping("{id}")
+    public String deleteBookById(@PathVariable Long id) {
+        bookService.deleteById(id);
+        return "Deleted a book with an id of " + id;
```

Comment:
let's not return any strings from this method. Simply annotate it with  @ResponseStatus(value = HttpStatus.NO_CONTENT)